scss/_debug.scss

```diff
@@ -0,0 +1,44 @@
+@if $enable-debugger-classes {
+  .#{$prefix}grid-debug {
+    &::before {
+      position: fixed;
+      top: 0;
+      left: 0;
+      z-index: 2000;
+      padding: $spacer * .5 $spacer;
```

Comment:
can this use em unit instead of rem unit for padding? Then it may adjust itself by its font size.